src/jit/unwind.cpp

```diff
@@ -153,6 +157,10 @@ void Compiler::unwindPushPopCFI(regNumber reg)
         // since it is pushed as a frame register.
         || (reg == REG_FPBASE)
 #endif // ETW_EBP_FRAMED
+#elif defined(_TARGET_ARM_)
+#if defined(_TARGET_UNIX_)
+        || generateCFIUnwindCodes()
```

Comment:
We should only come here if `generateCFIUnwindCodes` is true. I think this change is turning the whole condition into `if (true)`, no?

Reply:
@jkotas yes, you're right